cmd/demo/demo.go

```diff
@@ -101,7 +101,7 @@ func main() {
 		isTable, tableRendered := false, false
 		table := tablewriter.NewWriter(os.Stdout)
 
-		stream := sql.Run(slct, db)
+		stream := sql.Run(slct, db, "")
```

Comment:
Better to use "./" so it will be more meaningful

Reply:
"./" is also a director name, how about representing **don't save model on the local file system** using empty string "" ?

Reply:
That will be hacky too I think, since the argument is called `xxxDir` it should behave like a directory, "./" and "" should both mean current directory here. use "./" is more meaningful in the code.